src/current/molt/migrate-data-load-replicate-only.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Load and Replicate Separately
+summary: Learn how to migrate data from a PostgreSQL or MySQL database into a CockroachDB cluster.
+toc: true
+docs_area: migrate
+---
+
+Perform an initial bulk load of the source data using `data-load` mode, then use `replication-only` mode to replicate ongoing changes to the target.
+
+{{site.data.alerts.callout_success}}
+You can also [load and replicate in a single command]({% link molt/migrate-data-load-and-replication.md %}) using `data-load-and-replication`.
+{{site.data.alerts.end}}
+
+{% assign tab_names_html = "Load and replicate;Replicate separately" %}
+{% assign html_page_filenames = "migrate-data-load-and-replication.html;migrate-data-load-replicate-only.html" %}
+
+{% include filter-tabs.md tab_names=tab_names_html page_filenames=html_page_filenames page_folder="molt" %}
+
+{% include molt/molt-setup.md %}
+
+## Load data into CockroachDB
+
+Perform the initial load of the source data.
+
+1. Issue the [MOLT Fetch]({% link molt/molt-fetch.md %}) command to move the source data to CockroachDB, specifying [`--mode data-load`]({% link molt/molt-fetch.md %}#fetch-mode) to perform a one-time data load. This example command passes the source and target connection strings [as environment variables](#secure-connections), writes [intermediate files](#intermediate-file-storage) to S3 storage, and uses the `truncate-if-exists` [table handling mode](#table-handling-mode) to truncate the target tables before loading data. It also limits the migration to a single schema and filters three specific tables to migrate. The [data load mode](#data-load-mode) defaults to `IMPORT INTO`.
+
+	<section class="filter-content" markdown="1" data-scope="postgres">
+	Specify a replication slot name with `--pglogical-replication-slot-name`. This is required for [replication in a subsequent step](#replicate-changes-to-cockroachdb).
+
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source $SOURCE \ 
+	--target $TARGET \
+	--schema-filter 'migration_schema' \
+	--table-filter 'employees|payments|orders' \
+	--bucket-path 's3://migration/data/cockroach' \
+	--table-handling truncate-if-exists \
+	--pglogical-replication-slot-name cdc_slot \
+	--mode data-load
+	~~~
+	</section>
+
+	<section class="filter-content" markdown="1" data-scope="mysql">
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source $SOURCE \
+	--target $TARGET \
+	--schema-filter 'migration_schema' \
+	--table-filter 'employees|payments|orders' \
+	--bucket-path 's3://migration/data/cockroach' \
+	--table-handling truncate-if-exists \
+	--mode data-load
+	~~~
+	</section>
```

Comment:
I would perhaps add a call out here that this command should output the GTID that you need to begin replication from below.